packages/next-admin/src/components/NextAdmin.tsx

```diff
@@ -60,15 +69,15 @@ export function NextAdmin({
   };
 
   return (
-    <ConfigProvider basePath={basePath}>
-      <NextNProgress color="#6366f1" />
+    <ConfigProvider basePath={basePath} isAppDir={isAppDir}>
+      {!isAppDir && <NextNProgress color="#6366f1" />}
```

Comment:
Not supported ?

Reply:
Currently not https://github.com/apal21/nextjs-progressbar/issues/86